src/vs/workbench/contrib/tasks/browser/task.contribution.ts

```diff
@@ -205,14 +205,25 @@ MenuRegistry.appendMenuItem(MenuId.MenubarTerminalMenu, {
 	when: TaskExecutionSupportedContext
 });
 
+MenuRegistry.appendMenuItem(MenuId.MenubarTerminalMenu, {
```

Comment:
@anthonykim1 I don't think we want this in a menu and I don't see it in one. Should this be removed?

Reply:
Yeah, good point. I got confused with the command palette one. 
I don't see it in the menu also